client/me/billing-history/upcoming-charges-table.jsx

```diff
@@ -25,20 +30,22 @@ module.exports = React.createClass( {
 			<TransactionsTable
 				transactions={ transactions }
 				initialFilter={ { date: { newest: 20 } } }
-				description={ function( transaction ) {
-					var site = this.props.sites.getSite( Number( transaction.blog_id ) );
-
-					if ( site ) {
-						return (
-							<div className="transaction-links">
-								<a href={ purchasesPaths.managePurchase( site.slug, transaction.id ) }>
-									{ this.translate( 'Manage Purchase' ) }
-								</a>
-							</div>
-						);
-					}
-				}.bind( this ) }
-			/>
+				emptyTableText={ emptyTableText }
+				transactionRenderer={ this.renderTransaction } />
 		);
+	},
+
+	renderTransaction: function( transaction ) {
+		var site = this.props.sites.getSite( Number( transaction.blog_id ) );
```

Comment:
If we are going to depend on this prop in a way which could cause an exception (`undefined` doesn't have property `getSite`) then we should declare it as required in `propTypes`

```js
import React, { PropTypes } from 'react';

propTypes: {
	site: PropTypes.func.isRequired
}
```